package/libs/wolfssl/Makefile

```diff
@@ -23,14 +23,83 @@ PKG_BUILD_PARALLEL:=1
 PKG_LICENSE:=GPL-2.0+
 PKG_CPE_ID:=cpe:/a:yassl:cyassl
 
+define symdep
+$(firstword $(subst :, ,$(1)))
+endef
+
+define symdep_upper
+$(call toupper,$(call symdep,$(1)))
+endef
+
+define confsym
+$(firstword $(subst $(comma), ,$(lastword $(subst :, ,$(1)))))
+endef
+
+define confsym_upper
+$(call toupper,$(call confsym,$(1)))
+endef
+
+# Define symbols here that will be grouped later in menuconfig
+WOLFSSL_GROUP_DEP_SYMBOLS:= \
+	aes asn ecc asm coding rsa tls13
+
+# Note:
+#   sym1|sym2|...|synN means that `sym1` enables/disables all sym2..symN symbols
+#   sym1:sym2 means that `sym2` depdends on `sym1`
+WOLFSSL_DEFAULT_ON_OPTS:= \
+	wpas,sha512,fortress,fastmath \
+	ocsp,ocspstapling,ocspstapling2 \
+	aes \
+	aes:aesgcm \
+	asn \
+	asn:ecc \
+	ecc:eccshamir \
+	ecc:supportedcurves \
+	rsa \
+	asm rng harden sha512 errorstrings oldtls memory dh coding \
+	md5 sha chacha poly1305 hashdrbg filesystem inline extended-master \
+	oldnames asyncthreads
+
+WOLFSSL_DEFAULT_OFF_OPTS:= \
+	aes:aesccm \
+	aes:aesctr \
+	aes:aescfb \
+	aes:aeskeywrap \
+	asm:armasm \
+	asm:intelasm \
+	asm:intelrand \
+	asm:aesni \
+	asm:fast-rsa \
+	coding:base16 \
+	coding:base64encode \
+	crl \
+	crl:crl-monitor \
+	ecc:ecccustcurves \
+	ecc:curve25519 \
+	ecc:ed25519 \
+	ecc:fpecc \
+	ecc:eccencrypt \
+	rsa:rsapss \
+	tls13 \
+	tls13:tls13-draft18 \
+	tls13:tls13-draft22 \
+	tls13:tls13-draft23 \
+	distro 32bit singlethreaded dtls postauth hrrcookie sctp mcast openssh \
+	nginx haproxy bump sniffer signal maxstrength leanpsk leantls \
+	lowresource bigcache hugecache smallcache savesession savecert writedup \
+	atomicuser pkcallbacks xilinx camellia md2 nullcipher ripemd blake2 \
+	sessioncerts keygen certgen certreq certext sep hkdf x963kdf dsa \
+	compkey psk tlsv10 sslv3 trackmemory des3 idea arc4 cmac xts \
+	webserver fips hc128 rabbit sha224 sha3 qsh sni maxfragment alpn \
+	truncatedhmac renegotiation-indication secure-renegotiation \
+	session-ticket tlsx earlydata pkcs7 ssh scep srp indef smallstack \
```

Comment:
cannot enable earlydata without enabling tls13 (configure.ac: 2646)
cannot enable earlydata without enabling session tickets and/or PSK (configure.ac: 2650)
pkcs7 needs: rsa, sha, and either aes or 3des (configure.ac: 3905-3917). 3des is always on (stunnel)
scep also needs either aes or 3des.
ssh depends on aesgcm, ecc, sha, and sha512 (configure.ac: 4049-4052).